RecoTauTag/RecoTau/interface/DeepTauScaling.h

```diff
@@ -724,6 +724,540 @@ namespace deep_tau {
 
     };  // end scalingParamsMap_v2p5
 
+    const std::map<std::pair<FeatureT, bool>, ScalingParams> scalingParamsMap_PhaseIIv2p5 = {
```

Comment:
It is a general comment to scaling of deepTau input features rather that for this particular development. It will be better to read the scaling parameters together with a corresponding model files, e.g. from a json file, than hard-code them for each new training. 
This comment is for reference for future updates of deepTau producer, FYI @kandrosov

Reply:
Indeed, having the scaling constants in json was in the todo list for v2p5 integration, but it was postponed to fit the release schedule. I don't know if we want to implement it in this PR or postpone it once more.

Reply:
This comment was mainly to remind that it is planned, but I agree that it will be better to address it in a dedicated PR. Could some of deepTau developers work on this?